scripts/documentation_provider.py

```diff
@@ -55,6 +55,7 @@ def _patch_case(self) -> None:
                 continue
             members = {}
             self.classes[clazz["name"]] = clazz
+            clazz["events"] = []
```

Comment:
lets do it like we do for the `members` variable. Initialise it after line 56 and set it in line 97.